cvat-ui/src/reducers/interfaces.ts

```diff
@@ -569,3 +569,8 @@ export interface CombinedState {
     shortcuts: ShortcutsState;
     review: ReviewState;
 }
+
+export enum TaskDimension {
+    DIM_3D = '3d',
```

Comment:
I see similar enums in cvat-core/src/enums.js. What is a reason to have them here as well?

Reply:
Also I see that in python code you use DimensionType. Probably it is better to have similar name here as well instead of TaskDimension.

Reply:
Made changes accordingly.

Reply:
Made changes accordingly.